core/server/data/import/data-importer.js

```diff
@@ -133,6 +133,12 @@ DataImporter.prototype.doImport = function (data) {
                         if (results) {
                             importResults = importResults.concat(results);
                         }
+
+                        return utils.importSubscribers(tableData.subscribers, t);
+                    }).then(function (results) {
+                        if (results) {
+                            importResults = importResults.concat(results);
+                        }
```

Comment:
There's no `return` of a promise here, will the next `.then` be called successfully?